tests/pds/pds_common.go

```diff
@@ -363,6 +381,21 @@ func CleanupDeployments(dsInstances []*pds.ModelsDeployment) {
 	}
 }
 
+func GetReplicaNodes(appVolume *volume.Volume) ([]string, error) {
```

Comment:
can we have description for this function.

Reply:
Done